lib/io.toit

```diff
@@ -437,27 +440,29 @@ abstract class Reader implements old-reader.Reader:
   skip-up-to delimiter/int --to/int?=null --throw-if-absent/bool=false -> int:
     skipped := 0
     while true:
-      if not buffered_ or buffered_.size == 0:
+      buffered := buffered_
```

Comment:
This doesn't work.
`more_` creates the buffered if it doesn't exist yet.

Reply:
Fixed.